### ISSUE TITLE
[FEATURE] Tag Browser

### ISSUE THREAD

**Issue by Eranziel** [Eranziel/foundryvtt-lancer#667]
Currently the only way to view all tags available in a Foundry world is to open the tag editor. There should be a way to view all of the available tags and their descriptions.

**Comment by sealtrowitz** [comment]
Would adding a tag item type and compendium be a reasonable solution, given that tags exist in `tags.json` of lancer-data and lcps?

**Comment by msprijatelj** [comment]
Ironically, Tags as Items was removed in order to prevent nested "items within items", if I recall correctly.

**Comment by Eranziel** [comment]
Yup. Compendium data can only be accessed asynchronously, which makes it a bit of a nightmare to depend on for preparing the data for most item types. 

Storing them in a setting is a lot more robust and easier to work with, aside from the UI issue.
